Xamarin.Forms.Platform.iOS/Renderers/SwitchRenderer.cs

```diff
@@ -27,13 +29,26 @@ protected override void OnElementChanged(ElementChangedEventArgs<Switch> e)
 					Control.ValueChanged += OnControlValueChanged;
 				}
 
+				defaultOnColor = Control.OnTintColor;
 				Control.On = Element.IsToggled;
 				e.NewElement.Toggled += OnElementToggled;
+				UpdateOnColor();
 			}
 
 			base.OnElementChanged(e);
 		}
 
+		private void UpdateOnColor()
```

Comment:
We do not use the private keyword as it is the default accessibility level in C#